app/src/androidTest/java/com/hedvig/app/feature/embark/screens/ZignSecScreen.kt

```diff
@@ -0,0 +1,10 @@
+package com.hedvig.app.feature.embark.screens
+
+import com.agoda.kakao.common.views.KView
+import com.agoda.kakao.screen.Screen
+import com.hedvig.app.R
+
+class ZignSecScreen: Screen<ZignSecScreen>() {
```

Comment:
When moving to using stubbed intents, we no longer need this screen-file I think 🙌